<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbcustomformular</class>
 <widget class="QWidget" name="dbcustomformular">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>316</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rBmost">
      <property name="text">
       <string>most</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBlowest">
      <property name="text">
       <string>lowest</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rBall">
      <property name="text">
       <string>all</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="cBRead">
      <property name="text">
       <string>read</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cBWrite">
      <property name="text">
       <string>write</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="cBUsers">
      <property name="text">
       <string>Users</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="cBFiles">
      <property name="text">
       <string>Files</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="cBShares">
      <property name="text">
       <string>Share</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="cBFilelength">
      <property name="text">
       <string>FileLength</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>212</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="sholimitlabel">
      <property name="text">
       <string>show limit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sBShowLimit"/>
    </item>
    <item>
     <widget class="QCheckBox" name="cBShowLimit">
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
